packages/commonwealth/client/scripts/views/components/component_kit/CWContentPage/CWContentPage.tsx

```diff
@@ -192,7 +192,7 @@ export const CWContentPage = ({
         authorAddress={author?.address}
         authorCommunityId={authorCommunityId}
         collaboratorsInfo={collaborators}
-        publishDate={moment(createdOrEditedDate)}
+        publishDate={moment(createdOrEditedDate, 'X')}
```

Comment:
let's add your OP comment here about why it works, maybe something like your note from OP -> `NOTE: If 'x' is passed instead of 'X' it will show "Published 54 years ago" again.`

Reply:
@mzparacha just added a comment. Good call.